### ISSUE TITLE
Visual glitch when rendering some SVG NFTs in mobile app

### ISSUE THREAD

**Issue by felixakiragreen** [rainbow-me/rainbow#2737]
A while ago I filled an issue about SVGs, and then it was fixed: https://github.com/rainbow-me/rainbow/issues/2040

But there seems to be a reversion. A user reached out noticing a problem with The Worm's NFT:
<img height="640" alt="Mobile Screen Shot" src="https://user-images.githubusercontent.com/1843672/146835523-c0c17c2e-bdf7-4065-94e3-e8f508736865.png">

I checked it out — there isn't any problem with the actual NFT.  And I can confirm that multiple of my own SVG NFTs have rendering glitches as well.

[Here is how they are supposed to look - rainbow.me](https://rainbow.me/0xc0D4A42dD3Cf5AC320D82E20A1285b50eFe26615?family=The%20Worm%20NFT&nft=0xacd3cf818efe8ddce84c585ddcb147c4c844d3b3_2)

It's weird — some work fine, some kind of work, some don't work at all.

<img height="640" alt="Mobile Screen Shot" src="https://user-images.githubusercontent.com/1843672/146836202-72bc2aff-d47b-4035-9efd-33905531aa94.PNG">

 Let me know if I can provide anymore useful information.

**Comment by welps** [comment]
Hi, sorry for the delay in response. This unfortunately broke with the introduction of #2566 which was done to resolve *other* SVG issues. IIRC the endgoal is to move to https://github.com/react-native-skia/react-native-skia once it hits a certain maturity, but as a temporary solution we are currently using Cloudinary here to transform SVGs to PNGs on the main screen. 

There are certain conditions that will cause cloudinary to output a jacked up looking image and it looks like we've missed another condition so I'll be looking to release a fix early next week for this.

**Comment by welps** [comment]
Hi @felixakiragreen , this issue was fixed on our end, BUT note this change is not yet reflected for all of your NFTs because Cloudinary caches transformations for 7 days. I purged Cloudinary of its stored transformations, but did not realize they don't invalidate their CDN when you do this so they're stuck in there (can't invalidate a transformation that has been deleted either!). 

Here is an example of some of your NFTs that were never cached by Cloudinary so they picked up the fix immediately. The rest should follow in a few days. 

<img src="https://user-images.githubusercontent.com/677680/147991491-0246ddfa-beed-4253-b4e2-3bb721b7d5f7.jpg" width=40% /> <img src="https://user-images.githubusercontent.com/677680/147991492-7b87f80c-ae95-4b3b-93b7-12e20204434b.jpg" width=40% />
